core/templates/base-components/loading-message.component.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for showing the loading screen with a message.
+ */
+
+angular.module('oppia').component('loadingMessage', {
+  template: require('./loading-message.component.html'),
```

Comment:
We do not use relative paths.

Reply:
Actually, this for the HTML file of the component which is always tightly linked to the ts files. It is kind of like a canon in angular where the template URLs are always relative as they are supposed to exist with the same name(except for extension) and in the same place where its corresponding ts file exists. For the ts files, it should be absolute but for the tightly-knit component and its HTML file, this should be relative. 

As you can see here in Angular guide: https://angular.io/tutorial/toh-pt3#what-changed
```
@Component({
  selector: 'app-hero-detail',
  templateUrl: './hero-detail.component.html',
  styleUrls: ['./hero-detail.component.css']
})
```
The template and the style URLs are relative because they are supposed to be in one place.

What are your thoughts on this @bansalnitish, @seanlip

Reply:
It is okay, we use relative paths for HTML directives in other places too.

Reply:
Thanks @srijanreddy98, sounds good.